unison-src/transcripts/help.output.md

```diff
@@ -683,15 +683,17 @@ scratch/main> help
   
   reset
   `reset #pvfd222s8n`         reset the current namespace to the
-                              causal `#pvfd222s8n`
-  `reset foo`                 reset the current namespace to
-                              that of the `foo` namespace.
-  `reset foo bar`             reset the namespace `bar` to that
-                              of the `foo` namespace.
+                              hash `#pvfd222s8n`
+  `reset foo`                 reset the current namespace to the
+                              state of the `foo` namespace.
   `reset #pvfd222s8n /topic`  reset the branch `topic` of the
                               current project to the causal
                               `#pvfd222s8n`.
   
+  If you make a mistake using reset, consult the `branch.reflog`
```

Comment:
With this addition, could you make `reflog` the default name for `branch.reflog`? 🙏